doc-site/docs/tutorials/notarized-tokens.md

```diff
@@ -0,0 +1,135 @@
+# Notarized Tokens
+
+In this tutorial, you’ll learn how to create and manage a basic token using the **Notarized Tokens (noto)** domain. Unlike the private storage example, these tokens can be transferred between nodes publicly, demonstrating how assets (e.g., “cash”) can be issued and tracked on the blockchain using Paladin.
+
+## Prerequisites
+
+Make sure you have:
+
+1. Completed the [Private Storage Tutorial](./private-storage.md).
+2. A **running Paladin network** with at least three nodes (Node1, Node2, Node3).
+
+---
+
+## Overview
+
+This tutorial will guide you through:
+
+1. **Deploying a Noto Token**: Use the `NotoFactory` to create a “cash token” with a specific notary.
+2. **Minting Tokens**: Issue tokens to a particular node’s account.
+3. **Transferring Tokens**: Send tokens between different nodes to simulate basic payments.
+
+You can find the complete example code in the [Paladin example repository](https://github.com/LF-Decentralized-Trust-labs/paladin/tree/main/example/notarized-tokens).
+
+---
+
+## Step 1: Deploy a Noto Token
+
+First, create a **Noto Factory** instance and deploy a new token. In this scenario, Node1 will act as both the notary (the entity allowed to mint tokens) and the initial recipient of the minted cash.
```

Comment:
The notary is not just the minting party. The notary is the party that will approve and submit every transaction related to this token.